openedx_webhooks/github_views.py

```diff
@@ -455,6 +460,23 @@ def github_contractor_pr_comment(pull_request):
     )
 
 
+def has_contractor_comment(pull_request, session=None):
+    me = github_whoami(session=session)
+    my_username = me["login"]
+    comment_url = "/repos/{repo}/issues/{num}/comments".format(
+        repo=pull_request["base"]["repo"]["full_name"].decode('utf-8'),
+        num=pull_request["number"],
+    )
+    for comment in paginated_get(comment_url, session=github):
+        # I only care about comments I made
+        if comment["user"]["login"] != my_username:
+            continue
+        magic_phrase = "It looks like you're a member of a company that does contract work for edX."
```

Comment:
I don't think there's a way to share the text, unfortunately. But we can write tests!